examples/configuration.rs

```diff
@@ -17,11 +17,11 @@ extern crate exonum_configuration;
 
 use exonum::helpers::fabric::NodeBuilder;
 
-use exonum_configuration::ConfigurationService;
+use exonum_configuration::ConfigurationServiceFactory;
 
 fn main() {
     exonum::helpers::init_logger().unwrap();
     NodeBuilder::new()
-        .with_service(Box::new(ConfigurationService::new()))
+        .with_service(Box::new(ConfigurationServiceFactory {}))
```

Comment:
`{}` is redundant.

Reply:
Not really. :smile: If a structure is declared with braces, then it cannot be created without them. This works in other direction though.

I'm going to fix it for consistency. :smile: